package.json

```diff
@@ -17,11 +17,12 @@
     "@emoji-mart/react": "^1.1.1",
     "@emotion/react": "^11.11.4",
     "@emotion/styled": "^11.11.5",
-    "@glific/flow-editor": "^1.19.3-8",
+    "@glific/flow-editor": "^1.26.3-1",
     "@lexical/react": "^0.13.1",
     "@mui/icons-material": "^5.15.15",
     "@mui/material": "^5.15.15",
     "@mui/x-date-pickers": "^7.3.1",
+    "@nyaruka/temba-components": "^0.89.0",
```

Comment:
I don't think this would work.

Even if you add it here it will only work for the dev server and not the production build, as it is dynamically loading from inside node_modules which will not be present when we do an actual build and host.
You can maybe verify by deploying this branch on staging.

Can you check how Rapidpro loads it and lets follow the same logic.

Reply:
so rapidpro uses v1.19,0, which we were using earlier so we cant do that,
i tried to deploy this on staging and it worked as expected you can test once here https://staging.glific.com/flow